br/cmd/br/restore.go

```diff
@@ -163,3 +184,17 @@ func newStreamRestoreCommand() *cobra.Command {
 	command.Hidden = true
 	return command
 }
+
+func newEBSMetaRestoreCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "ebs",
+		Short: "phase 1 of EBS-based restore to restore volumes and other info",
+		Args:  cobra.NoArgs,
+		RunE: func(command *cobra.Command, _ []string) error {
+			return runEBSMetaRestoreCommand(command, task.EBSMetaRestoreCmd)
+		},
+	}
+	task.DefineRestoreEBSMetaFlags(command)
+	command.Hidden = true
```

Comment:
why hidden?🤔

Reply:
those commands are used by tidb-operator only now, don't want to expose to user